src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,108 @@ tags: runtime-traditionalweb;
 summary: Wizard splits complex tasks and process into steps.
 ---
 
-# Wizard
+# Wizard UI Pattern
+You can use the Wizard UI pattern to split large complex tasks and processes into smaller manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
 
-Split complex tasks and process into steps.
+**How to Use the Wizard UI Pattern**
 
-Use the Wizard to present numbered steps or conditions that the end user needs to complete or reach a goal. Use it to split large tasks into more manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
+1. In Service Studio, in the Toolbox, search for  `Wizard`. 
 
-**How to use**
+    The Wizard widget is displayed.
+    
+     ![](<images/wizard-image-23.png>)
+
+1. From the Toolbox, drag the Wizard widget onto your application's screen.
 
-1. Drag Wizard pattern into the preview.
+    By default, the Wizard widget contains three Wizard Item blocks. Each Wizard Item represents a step. You can add or delete Wizard Items as required.
 
     ![](<images/wizard-image-1.png>)
 
-1. Drag as many WizardItems in the Content placeholder as required. 
+**Create a Wizard with Navigation Buttons**
 
-1. Set the content in the placeholders.
+The following example demonstrates how you can create a three step Wizard with navigation buttons.
```

Comment:
If 3 steps is the default, will they easily be able to create a Wizard with more navigation buttons? Would a 4 step wizard (to add an item there) be more helpful? @pacferreira @MiguelDomingues ?

Reply:
Would a 4 step wizard make the procedure (document) too long

Reply:
Unsure, but if people want to add a step, how do they do it? We never explain that in this document. It just feels awfully convenient that we're using the exact number of steps that the widget comes with :) At least a note around here somewhere saying: "If you want to add more steps, <do this>. If you want to remove, <do this>. Adjust the example steps accordingly to the new number of steps you have." or similar.

Reply:
I've added an extra step adding an extra Wizard Item